### ISSUE TITLE
Bug: Wordpress Databse Error

### ISSUE THREAD

**Issue by mateofaivre** [msaari/relevanssi#64]
Hi,

When i go to the indexing tab after installing Relevanssi, I encounter this error :
```
WordPress database error: [BLOB, TEXT, GEOMETRY or JSON column 'customfield_detail' can't have a default value]
ALTER TABLE ga_relevanssi ALTER COLUMN `customfield_detail` SET DEFAULT ''
WordPress database error: [BLOB, TEXT, GEOMETRY or JSON column 'mysqlcolumn_detail' can't have a default value]
ALTER TABLE ga_relevanssi ALTER COLUMN `mysqlcolumn_detail` SET DEFAULT ''
```

I can't index anything. I tried to downgrade Relevanssi 4.23.1, but it doesn't change.
I'm on WP 6.7.1, clean install, Twenty Twenty-Five theme.

Thank you

**Comment by msaari** [comment]
I find it weird this is a problem now – those default values were added four years ago. You'd have to downgrade to 2.15 or so. I have already removed those default values, I'll release a new version tomorrow.

**Comment by msaari** [comment]
I didn't see any errors with this, but it's possible it's just suppressed. In any case, I just released 4.24.1 that removes the default values, so there shouldn't be any errors from that now.
